#new pose
float64 x
float64 y
float64 theta

---
#old pose
float64 x
float64 y
float64 theta
